evalai/challenges.py

```diff
@@ -21,6 +28,16 @@ def challenges(ctx):
         echo(welcome_text)
 
 
+@click.group()
+@click.pass_context
+@click.argument('CHALLENGE', type=int)
+def challenge(ctx, challenge):
+    """
+    Displays challenge specific details.
+    """
+    ctx.obj = Challenge(challenge)
```

Comment:
It's actually in the child commands, but I thought this would make sense when it's seen in `--help`. I'll change it to `Challenge related operations.`